### PR TITLE
Tut2, plotBelief, and another install comment

### PR DESCRIPTION
**Pull request by dehann**
xref
- #40 
- #41

### CODE REVIEW

**Comment by dehann** [comment]
I'm not sure how else to fix the missing packages IIF or GraphPlot the first time the BInder notebook loads.

**Comment by dehann** [comment]
This is using RoMEPlotting v0.9.0.

I'd suggest we switch to plotBelief over plotKDE.  It's the same function with a better signature:
- https://github.com/JuliaRobotics/Caesar.jl/issues/806
  - https://github.com/JuliaRobotics/Caesar.jl/discussions/811

**Comment by jim-hill-r** [comment]
Can you not run import twice? What's the big deal about leaving it uncommented?

**Comment by dehann** [comment]
think we can leave it uncommented, let me try that quick.

**Comment by dehann** [comment]
Lets leave it uncommented.

Screenshot from 2022-03-26 12-33-53

File: julia/caesar-api/icra-2-nongaussian.ipynb [hunk 1]
```diff
@@ -38,10 +38,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Install if needed with: \n",
     "# import Pkg; Pkg.add(Pkg.PackageSpec(name=\"IncrementalInference\", version=\"0.27\"))\n",
```
Comment (dehann) [inline julia/caesar-api/icra-2-nongaussian.ipynb]:
```suggestion
    "import Pkg; Pkg.add(Pkg.PackageSpec(name=\"IncrementalInference\", version=\"0.27\"))\n",
```

**Pull request merged**